src/features/preview.ts

```diff
@@ -433,25 +433,33 @@ export class AsciidocPreview {
     vscode.workspace.openTextDocument(this._resource).then(vscode.window.showTextDocument)
   }
 
-  private async onDidClickPreviewLink (href: string) {
-    let [hrefPath, fragment] = decodeURIComponent(href).split('#')
-    // From Markdown plugin
-    if (hrefPath[0] !== '/') {
+  private resolveDocumentLink (href: string): { hrefPath: string, fragment: string } {
+    let [hrefPath, fragment] = href.split('#').map((c) => decodeURIComponent(c))
+    if (hrefPath.startsWith('file://')) {
```

Comment:
This is not necessarily an absolute path. For instance:

- `file:///path/to/dir` is an absolute path
- `file://../../path/to/dir` is a relative path

Both paths start with `file://` but only the first one is an absolute path. I think we should remove the protocol `file://`.